### PR TITLE
added scroll to contact us and email validation to subscribe us

### PR DESCRIPTION
**Pull request by jainmudit68**
in reference to issue #101 

i have added the scroll to navigate to contact us/subscribe place and email validation (client-side) to the input box.

image

### CODE REVIEW

File: components/homepage/About/About.jsx [hunk 1]
```diff
@@ -62,4 +62,4 @@ const About = () => {
   );
 };
 
-export default About;
+export default About;
```
Comment (kaustubhai) [inline components/homepage/About/About.jsx]: revert this file change

**Review by kaustubhai — CHANGES_REQUESTED** [review]

File: components/homepage/About/About.jsx (hunk 1 above)
Comment (jainmudit68) [reply components/homepage/About/About.jsx]: i am sorry i didn't get it! what to revert in this? this is the same file currently present in the original repository..

I think i couldn't follow what you mean? could you please explain

Comment (kaustubhai) [reply components/homepage/About/About.jsx]: I think you have removed the next empty line. If you are unable to see the changes run following in sequence:
1. `git reset HEAD~`
2. `git add components/homepage/Footer/Footer.jsx`
3. `git add components/Navbar/Navbar.jsx`
4. `git commit -m '<MESSAGE>'`
5. `git push origin <BRANCH_NAME>`

Happy to help more if required

**Comment by jainmudit68** [comment]
i have opened a new pull request clearing all the code and getting everything done again as PR #126 
(after cloning the code again i found that the button for contact us section was removed so i added it too)

please review it, thank you. @kaustubhai

**Pull request closed without merging**